certified-connectors/Nodefusion Portal/Readme.md

```diff
@@ -0,0 +1,33 @@
+
+## NodefusionPortal Connector
+This is a Nodefusion Portal connector. The connector is published here with the intent to provide Nodefusion customers with the actions they already have in Nodefusion Portal so they can automate its processes.
+

```

Comment:
@hkusulja Please refer the template for readme.md file [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates) and add the required sections.